charts/kube-master/values.yaml

```diff
@@ -80,3 +80,51 @@ scheduler:
       memory: 512Mi
 
 revisionHistoryLimit: 3
+
+dex:
+  enabled: false
+  image:
```

Comment:
Please move the image values to to the `images.yaml` https://github.com/sapcc/kubernikus/blob/master/charts/images.yaml#L2-L9

To properly reference it again we need this (arguably ugly) helper methods to: 
https://github.com/sapcc/kubernikus/blob/master/charts/kube-master/templates/_helpers.tpl#L31-L38

The advantage of this is that we can easily update the image versions at a central place. Also it's possible to configure different versions for different Kubernetes releases. Could be useful for the Dashboard with `v1.10.1` as default and `v2.0.0-beta4` for pre-release version.

Reply:
fixed by: https://github.com/sapcc/kubernikus/pull/464/commits/ab997d3abb599ada9b7e84e652f4b8b80111e944#diff-d238b1eb95a4202a4d4d6b0ca52509dc